code/modules/food_and_drinks/food/snacks_frozen.dm

```diff
@@ -43,6 +43,143 @@
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_caramel"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/caramel = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/caramel = 10)
+	filling_color = "#BC762F"
+	tastes = list("ice cream" = 1, "caramel" = 1)
+	foodtype = DAIRY | SUGAR
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/banana
+	name = "banana ice cream scoop"
+	desc = "The ice cream of choice for clowns everywhere. Honk!"
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_banana"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/banana = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/banana = 10)
+	filling_color = "#DEDE00"
+	tastes = list("ice cream" = 1, "banana" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/orange_creamsicle
+	name = "orange creamsicle scoop"
+	desc = "An ice cream flavor made after a popular popsicle flavor. It is not quite the same off the stick..."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_orangesicle"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/orange_creamsicle = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/orange_creamsicle = 10)
+	filling_color = "#D8B258"
+	tastes = list("ice cream" = 1, "oranges" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/peach
+	name = "peach ice cream scoop"
+	desc = "Ice cream flavored with peaches; it is rather uncommon due to wizards buying up most of it."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_peach"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/peach = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/peach = 10)
+	filling_color = "#CD8D68"
+	tastes = list("ice cream" = 1, "peaches" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/cherry_chocolate
+	name = "cherry chocolate ice cream scoop"
+	desc = "A wonderfully tangy and sweet ice cream made with coco and cherries."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_cherry chocolate"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/cherry_chocolate = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/cherry_chocolate = 10)
+	filling_color = "#6F0000"
+	tastes = list("ice cream" = 1, "cherries" = 1, "chocolate" = 1)
+	foodtype = DAIRY | SUGAR | FRUIT | CHOCOLATE
+
+/obj/item/reagent_containers/food/snacks/ice_cream_scoop/meat
+	name = "meat lover's ice cream scoop"
+	desc = "Ice cream flavored with meat, because someone wanted meat in their ice cream."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "icecream_mob"
+	bonus_reagents = list(/datum/reagent/consumable/ice_cream/meat = 2)
+	list_reagents = list(/datum/reagent/consumable/ice_cream/meat = 10)
+	filling_color = "#BD0000"
+	tastes = list("ice cream" = 1, "blood" = 1)
+	foodtype = DAIRY | SUGAR | MICE
```

Comment:
uh... right...

Reply:
Ice cream vat had an unused meat flavor, so I added it in with the rest.